swap_meet/vendor.py

```diff
@@ -1,2 +1,114 @@
+from .item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    def remove(self, item):  
```

Comment:
Small detail: it's nice to see consistent white space between functions, for readability :)